data/sectionsData.js

```diff
@@ -1,17 +1,8 @@
-import levelsData from 'data/levelsData';
-import 'core-js/actual/array/group-by';
+import levelsData from "data/levelsData";
+import "core-js/actual/array/group-by";
 
-const groupBySection =
-  levelsData.groupBy(
-    (level) => level.section
-  );
+const sectionsData = levelsData.groupBy((level) => level.section);
 
-let sectionsData = [];
-sectionsData.push(groupBySection);
-
```

Comment:
you didnt need to push sectionsData into another array, sectionsData is already an array